packages/phone_log/.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
+.atom/
+.idea
+.packages
+.dart_tool/
+.pub/
+build/
+ios/.generated/
+packages
+pubspec.lock
```

Comment:
You don't need this file. I just updated the .gitignore file at the root. That should be enough.